core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -31,7 +31,7 @@ import org.scalatest.Assertions._
 import org.scalatest.PrivateMethodTester
 import org.scalatest.concurrent.Eventually
 
-import org.apache.spark._
+import org.apache.spark.{FakeSchedulerBackend => _, _}
```

Comment:
LGTM. Maybe it would have been better if we leave a comment so people don't mistakenly remove for the time being :-).

Reply:
Thanks. Ya. It's a general problem in Scala 2.13 because we don't have any CIs. Not only tests, Scala 2.13 compilation itself is already broken in `sql/catalyst`. It's sad that there is no way to protect properly. :(

Maybe, we had better consider enable Scala 2.13 CI partially for compilation and testing.

WDYT, @HyukjinKwon and @srowen ?

Reply:
I haven't tested it by myself yet so I don't know the stability about 2.13 yet but +1 for the idea.